packages/nimble-components/src/table/specs/table-column-specs/table-column-elapsed-time-hld.md

```diff
@@ -0,0 +1,177 @@
+# Elapsed Time Table Column
+
+## Overview
+
+The elapsed time table column will provide a way to visualize numeric data that represents a timespan. A 'timespan' is simply a numeric value representing a total number of seconds (i.e. a value of 5 means '5 seconds').
+
+### Background
+
+[Work Item](https://ni.visualstudio.com/DevCentral/_workitems/edit/2526116)
+
+### Non-goals
+
+-   APIs to configure individual aspects of the display (i.e. fractional digits for seconds display)
+-   APIs to format the elapsed time in a variety of ways
+    -   Show elapsed time as digital display (00:01:20.323)
+    -   Show elapsed time with symbols indicating a quantitative value (i.e. "< 1 sec")
+
+### Risks and Challenges
+
+Formatting the elapsed time units in the exact way that SLE presents them would require custom logic that differs from what the browser provides natively using APIs like `Intl.Numberformat.format()`. The preference here is to align with current browser behaviors for unit formatting, which includes localization.
```

Comment:
I asked a similar question on the file size HLD: can / should we expose an API that clients can use to format time in the same way as the column does without instantiating a column? Ideally the `ElapsedTimePipe` would have the same formatting logic as the elapsed time column.